lib/wickedbase.pm

```diff
@@ -381,15 +382,10 @@ Used to syncronize the wicked tests for SUT and REF creating the corresponding m
 
 =cut
 sub do_mutex {
```

Comment:
we do not need the `$self->{wicked_need_sync}`  anymore, cause all suite act with REF<->SUT sync